Server/src/Server.Infrastructure/DomainServices/IdentityDomainService.cs

```diff
@@ -47,7 +49,7 @@ public IdentityDomainService(IIdentityRepository identityRepository, ISecurityTo
         CancellationToken cancellationToken)
     {
         var identity = await _identityRepository.FirstOrDefaultByEmailAsync(email, cancellationToken);
-        if (identity == default)
+        if (identity == default || !identity.Roles.Contains(Role.User.GetDisplayName()))
```

Comment:
to idąc tą logiką, nic nie powstrzyma mnie na razie przed zalogowaniem się jako pracownik i wejściem na normalną stronę jakby nic się nie stało